clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1256,34 +1259,19 @@ def create_file(image, modality, bids_dir, mod_to_update):
         return nan
 
 
-def viscode_to_session(viscode):
-    """Replace the session label 'bl' with 'M00' or capitalize the session name passed as input.
-
-    Args:
-        viscode: session name
-
-    Returns:
-        M00 if is the baseline session or the original session name capitalized
-    """
-    if viscode == "bl" or viscode == "m0":
-        return "ses-M00"
-    else:
-        return "ses-" + viscode.capitalize()
-
-
 def session_to_viscode(session_name):
-    """Replace the session label 'bl' with 'M00' or capitalize the session name passed as input.
+    """Replace the session name passed as input with the session label 'bl' or 'mXXX'.
 
     Args:
-        session_name: MXX
+        session_name: MXXX
 
     Returns:
-        M00 if is the baseline session or the original session name capitalized
+        M000 if is the baseline session or the original session name capitalized
```

Comment:
To be converted to numpydoc?

Reply:
Well, converting all the docstrings impacted by this PR would take a lot of time, but for this one why not